src/renderer/widgets/AddProxyModal/lib/add-proxy-utils.ts

```diff
@@ -0,0 +1,35 @@
+import { Step, TxWrappers } from './types';
+
+export const addProxyUtils = {
+  isInitStep,
+  isConfirmStep,
+  isSignStep,
+  isSubmitStep,
+
+  hasMultisig,
+  hasProxy,
+};
+
+function isInitStep(step: Step): boolean {
```

Comment:
seems that it's not only proxy steps.

Reply:
this file contains utility functions only for `AddProxyModal` manager